token-swap/program/src/processor.rs

```diff
@@ -553,27 +555,29 @@ mod tests {
         amount: u64,
     ) -> ((Pubkey, Account), (Pubkey, Account)) {
         let token_key = pubkey_rand();
-        let mut token_account = Account::new(0, size_of::<SplMint>(), &program_id);
+        let mut token_account = Account::new(0, <SplMint as Pack>::LEN, &program_id);
```

Comment:
`SplMint::get_packed_len()`?

Reply:
Oh! Ha!  I meant to go looking for something less awkward after I got it working..  Thanks!

Reply:
Done!